mm2src/mm2_main/src/lp_swap.rs

```diff
@@ -234,6 +282,7 @@ pub async fn process_msg(ctx: MmArc, topic: &str, msg: &[u8]) {
                 SwapMsg::TakerFee(taker_fee) => msg_store.taker_fee = Some(taker_fee),
                 SwapMsg::MakerPayment(maker_payment) => msg_store.maker_payment = Some(maker_payment),
                 SwapMsg::TakerPayment(taker_payment) => msg_store.taker_payment = Some(taker_payment),
+                _ => (),
```

Comment:
Even though you're going to move `SwapMsg::WatcherMessage` to a separate enum, but I'd like to warning you with using `_` pattern at the match control flow.
If someone add a new `SwapMsg` later, this code will compile, but the programmer may forget to check the new variant here.